registrations/tests.py

```diff
@@ -2473,20 +2475,28 @@ def test_registration_process_clinic_good(self):
         # . check number of calls made:
         #   message set, schedule, jembi registration, id_store mother,
         #   id_store mother_reverse, id_store registrant_rever, id_store patch
-        self.assertEqual(len(responses.calls), 7)
+        self.assertEqual(len(responses.calls), 8)
```

Comment:
Possibly a nitpick, not sure about this codebase. I think the 7 calls are listed in the comment above, should the 8th one be added somewhere in that list?